lib/tasks/jenkins.rake

```diff
@@ -0,0 +1,23 @@
+begin
+  require 'English'
+rescue LoadError
+else
+  namespace :jenkins do
+    task :unit => ['jenkins:setup:minitest', 'rake:test']
+
+    namespace :setup do
+      task :minitest do
+        ENV['MINITEST_REPORTER'] = 'JUnitReporter'
+        ENV['MINITEST_REPORTERS_REPORTS_DIR'] = 'jenkins/reports/unit'
```

Comment:
I'd prefer to set this in Jenkins via env vars:
https://github.com/theforeman/jenkins-jobs/blob/1041458b9bb8e68e53b8d863e0ff251540632304/theforeman.org/pipelines/test/smart-proxy.groovy#L98-L100

Reply:
That would make this task irrelevant, which is fine, but it does mean that it's harder to just replicate this locally. As in, a user has to jump through more hoops rather than just running a nicely encapsulated rake task.

Thoughts?